docs/overrides.md

```diff
@@ -34,3 +35,36 @@ GLOBAL FLAGS
 DESCRIPTION
   Clear Overrides for a given Feature or Project.
 ```
+
+## `dvc overrides update`
+
+Update an override
```

Comment:
did you run `yarn build`? how come this didn't update with your changes? 🤔

Reply:
i did not..